src/includes/WcProxy/ThemeProxy.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * Theme proxy.
+ *
+ * @package Alma\Woocommerce\WcProxy
+ */
+
+namespace Alma\Woocommerce\WcProxy;
+
+use WP_Theme;
+
+/**
+ * Theme proxy.
+ */
+class ThemeProxy {
+
+	/**
+	 * Get theme data.
+	 *
+	 * @return WP_Theme
+	 */
+	public function get_theme() {
```

Comment:
I think if you are in a Proxy, you can make this method private and use only proxy methode to get data ? Why do you think ?

Reply:
Done